packages/data-transport-layer/.gitignore

```diff
@@ -0,0 +1 @@
+db
```

Comment:
Why is this here? It's not obvious from the commit message

Reply:
Can pull to the top level gitignore

Reply:
Oh I see, it is due to the DTL creating this folder. I'll add a PR for that.

Reply:
https://github.com/ethereum-optimism/optimism/pull/487

Reply:
^updated this in the PR since there was an extra `contracts` in the path